### PR TITLE
changes by Faith 12:52pm

### PR DESCRIPTION
**Pull request by fwallacephd**
Updated who will narrate.
Added another line to the story.

### CODE REVIEW

**Comment by airportyh** [comment]
This is your first pull request!

File: chapter_1.py [hunk 1]
```diff
@@ -1,8 +1,2 @@
 print("One late Thursday morning, as they gathered at the haunted pizza place,")
-
-
-But let me tell you a story,  which you might not believe. We will codes,  letters and numbers
-gives us characters which may set our mind at ease..... or maybe ...........
-
-'HI THERE' .. yes you .... you with the nostril hair..... hhmmmm... yes you.... Deadpool here..
-Nice novel so far, right ?
+print ("There was a skull sitting on the counter, humming.")
```
Comment (airportyh) [inline chapter_1.py]: Scary!

Comment (airportyh) [reply chapter_1.py]: I am going to try starting a review.

Comment (airportyh) [reply chapter_1.py]: This is a continuation of a sentence and should not be capitalized.

**Comment by fwallacephd** [comment]
Made the changes requested.

File: chapter_1.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
 print("One late Thursday morning, as they gathered at the haunted pizza place,")
-print ("There was a skull sitting on the counter, humming.")
+print ("there was a skull sitting on the counter, humming.")
```
Comment (Nerajno) [inline chapter_1.py]: Is this all your literary acumen has ?

**Pull request merged**